src/components/modal/ModalComposite.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import * as _ from 'underscore';
+import { IModalProps, Modal } from './Modal';
+import { IModalHeaderProps, ModalHeader } from './ModalHeader';
+import { IModalFooterProps, ModalFooter } from './ModalFooter';
+import { IModalBackdropProps, ModalBackdrop } from './ModalBackdrop';
+import { IReduxStatePossibleProps } from '../../utils/ReduxUtils';
+import { ModalBackdropConnected } from './ModalBackdropConnected';
+import { ModalHeaderConnected } from './ModalHeaderConnected';
+import { JSXRenderable } from '../../utils/JSXUtils';
+import { ModalConnected } from './ModalConnected';
+import { ModalBody } from './ModalBody';
+
+export interface IModalCompositeOwnProps extends IModalProps, IModalHeaderProps, IModalFooterProps, IModalBackdropProps {
+  modalHeaderChildren?: JSXRenderable;
+  modalHeaderClasses?: string[];
```

Comment:
as-tu considerer mettre `IClassName` ici ? j'ai ajoute ca dans une PR recemment.

Reply:
I could probably change it to use it. For now the components underneath do not use classnames. They are built manually.